HeartbeatLogging/Sources/HeartbeatsPayload.swift

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 import Foundation
+import GoogleUtilities_NSData
```

Comment:
Weird. When I command click on the import in Xcode, it brings me to the swift interface. Yet, it still fails to build with `no such module GoogleUtilities_NSData`